src/core/Song.cpp

```diff
@@ -467,13 +468,30 @@ void Song::processAutomations(const TrackList &tracklist, MidiTime timeStart, fp
 		}
 	}
 
+	if (!m_oldAutomatedValues.isEmpty())
```

Comment:
```suggestion
	// Checks if an automated model stopped being automated by automation patterns
	// so we can move the control back to any connected controller again
	if (!m_oldAutomatedValues.isEmpty())
```

Just some comments for reference

Reply:
I will commit it